src/_actions/TradingOptionsActions.js

```diff
@@ -7,12 +7,17 @@ export const updateTradingOptions = (symbol, opts) => ({
     opts,
 });
 
-export const getTradingOptions = symbol =>
+export const getTradingOptions = (symbol, cb) =>
```

Comment:
cb = callback, or something like onDone, cb => bad name ;)
Do this as parameter cb = () => {}
Then you will not need to check for existing,